commons-vfs2/src/main/java/org/apache/commons/vfs2/util/MonitorInputStream.java

```diff
@@ -34,6 +34,10 @@ public MonitorInputStream(final InputStream in) {
         super(in);
     }
 
+    public MonitorInputStream(final InputStream in, final int size) {
```

Comment:
Please add a Javadoc comment for this public method.

Reply:
This is and the other one are done.